packages/api/lib/ems.js

```diff
@@ -0,0 +1,280 @@
+'use strict';
+
+const { get } = require('lodash');
+const fs = require('fs');
+const moment = require('moment');
+const os = require('os');
+const path = require('path');
+const aws = require('@cumulus/common/aws');
+const log = require('@cumulus/common/log');
+
+const { deconstructCollectionId } = require('../es/indexer');
+const { Search, defaultIndexAlias } = require('../es/search');
+
+/**
+ * This module provides functionalities to generate EMS reports.
+ * The following environment variables are used:
+ * process.env.ES_SCROLL_SIZE: default to 1000
+ * process.env.ES_INDEX: set for testing purpose, default to fTOdefaultIndexAlias
+ * process.env.ems_provider: default to 'cumulus'
+ * process.env.stackName
+ * process.env.bucket
```

Comment:
What is "bucket" used for?

Reply:
store the ems reports

Reply:
Please update the comment to indicate that.